### DISCUSSION TITLE
Remove Authme Support

### DISCUSSION THREAD

**Discussion by CqldFire** [NamelessMC/Nameless#2592] [General]
### Describe the feature

In my opinion, authme should be removed because no one uses it and it's annoying for the developers (like me) to maintain.

Authme requires users to change their skin, which is a hassle, so no one really wants to do that and it can be a deterrent for registering with a website.

**Comment by TeemoCell** [comment]
I agree. I don't know anyone who uses authme.

**Comment by samerton** [comment]
Ideally, both AuthMe (which is integration with the plugin) and MCAssoc (skin changing mechanism) should both be moved to separate modules.

However, the core does not allow for this yet, so we'll have to decide what to do about this.

**Comment by itsmemac** [comment]
Do move authme to module rather than completely removing it from NMC since we used it in the past for easy login
as some were not upto creating another account and this was a sleek feature where when they play, they login with authme (Just extra protective layer for MC server since more accounts are being stolen ATM so to avoid damange in those instance, we had authme) and the same account is used in website.
Made things far easy

**Comment by MotikaCraft-TheJoshua** [comment]
i think that authme is the standard for cracked mc, and cracked mc is > 50% of Minecraft itself (mojang don't care about cracked MC)

I think that AuthMe Integration should be kept within core of NMC

If you don't want to cover it while doing your own development on something, just don't do anything related to it, i am sure you can use default code for authme parts and edit everything else other than that, it's a fact that authme is standard for any cracked MC server and will always be, it even now received Velocity support (bungee support was added a lot ago)...
